### PR TITLE
docs: UI analysis partially done

### PR DESCRIPTION
**Pull request by Bookweight**
## Types of changes
**Thanks for sending a pull request! Please fill in the following content to let us know better about this change.**
Please put an `x` in the box that applies

- [ ] **Bugfix**
- [ ] **New feature**
- [ ] **Refactoring**
- [ ] **Breaking change** (any change that would cause existing functionality to not work as expected)
- [x] **Documentation Update**
- [ ] **Other (please describe)**

## Description
Add to chapter 6: UI Analysis, more UI image needed.
## Related Issue
Advises on document design needed.

### CODE REVIEW

**Review by asas1asas200 — CHANGES_REQUESTED** [review]
Can you give `doc/img/S__2588679.jpg` a reasonable name.

And add a record to `版次變更紀錄`.

File: doc/SRS.tex [hunk 1]
```diff
@@ -149,4 +150,23 @@ \subsection{系統特色}
...
+\newpage
+\subsection*{三.教室借用介面:}
+\bigskip
+\subsubsection*{1.樓層選擇:可選擇系館樓層}
+\bigskip
+\subsubsection*{2.登入狀態:可按此登入、登出}
+\bigskip
+\subsubsection*{3.日期選擇:可選擇欲借用之日期}
+\bigskip
+\subsubsection*{4.教室狀態:直行為時間、橫列為教室，教室狀態有:已有固定課程、已經借用、尚未歸還鑰匙、可借用}
```
Comment (asas1asas200) [inline doc/SRS.tex:168]: Why did you write number manually rarather than use `\subsection` without `*`?

Comment (asas1asas200) [reply doc/SRS.tex:168]: Please use fullwidth `：` instead `:`.

Comment (Bookweight) [reply doc/SRS.tex:168]: ok i'll try, it seems bettter that way

File: doc/SRS.tex [hunk 2]
```diff
@@ -25,6 +25,7 @@
 \graphicspath{{./img/}}
 
 \usepackage{indentfirst}
+\usepackage{wrapfig}
```
Comment (asas1asas200) [inline doc/SRS.tex:28]: This package is not in use.
It can be removed.

**Review by asas1asas200 — CHANGES_REQUESTED** [review]

File: doc/SRS.tex [hunk 3]
```diff
@@ -149,4 +149,33 @@ \subsection{系統特色}
...
+\begin{minipage}{0.4\linewidth}
+	\subsubsection*{\textcolor{red}{1.}樓層選擇：}可選擇系館樓層
+	\subsubsection*{\textcolor{red}{2.}登入狀態：}可按此登入、登出
+	\subsubsection*{\textcolor{red}{3.}日期選擇：}可選擇欲借用之日期
+	\subsubsection*{\textcolor{red}{4.}教室狀態：}直行為時間、橫列為教室，教室狀態有：已有固定課程、已經借用、尚未歸還鑰匙、可借用
+\end{minipage}
+
+\end{document}
+
+\textcolor{red}{1.}
```
Comment (asas1asas200) [inline doc/SRS.tex:181]: Put this on top like `LightGray` then this PR can be merged.

**Review by asas1asas200 — CHANGES_REQUESTED** [review]

File: doc/SRS.tex (hunk 3 above)
Comment (Bookweight) [reply doc/SRS.tex:181]: that line is added by accident ,you can delete it

**Comment by asas1asas200** [comment]
Merged manually.
Close this PR.

**Pull request closed without merging**